docs/src/user-guide/quick-start/setup/main-page.md

```
@@ -0,0 +1,131 @@
+
+The unstructured text flavour of CLP is appropriate for unstructured text logs where each log event contains a
+timestamp and may span one or more lines.
+
+:::{note}
+If your logs do not contain timestamps or CLP can't automatically parse the timestamps in your logs,
+it will treat each line as an independent log event.
+:::
+
+For example:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)